2-mandatory/6-writing-tests-advanced.js

```diff
@@ -89,3 +124,12 @@ function formatCourseworkResult(trainee) {
     subjects: ["HTML", "CSS", "Databases"]
   }
 */
```

Comment:
These tests look good - but have a think about the best way to name the tests.
Imagine we make a code change which makes this test fail. In the test output - we will see `Trainee Aman, score of ` but this doesn't really tell us which part of the code is broken. Maybe a better name for this is `should return error when coursework score is missing` (just an example).